app-management/guides/About App Management.md

```diff
@@ -1,3 +1,8 @@
 # About App Management
-Our App Management APIs provide your app with access to basic user data and app functionality. 
-  
+Wix's App Management APIs provide your app with access to basic user data and specific app functionality. 
+
+- The [App Instance API](https://dev.wix.com/api/rest/app-management/apps/app-instance/introduction) exposes basic data about the site and your app installed in it.
+- The [Billing API](https://dev.wix.com/api/rest/app-management/apps/billing/introduction) enables your app to lead customers seamlessly into Wix's checkout process from your platform.
+- The [Embedded Scripts API](https://dev.wix.com/api/rest/app-management/apps/embedded-scripts/introduction) enables your app to inject custom script tags to the website's head tag.
```

Comment:
Consider:

"inject into the website's \<head> tag."

Reply:
done